app/upgrades/v4/upgrades.go

```diff
@@ -5,16 +5,21 @@ import (
 	"github.com/classic-terra/core/app/upgrades"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/staking/keeper"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 )
 
 func CreateV4UpgradeHandler(
 	mm *module.Manager,
 	cfg module.Configurator,
 	_ upgrades.BaseAppParamManager,
-	_ *keepers.AppKeepers,
+	keepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		// to run staking store migration
+		stakingKeeper := keepers.StakingKeeper
+		stkingMigrator := keeper.NewMigrator(stakingKeeper)
+		stkingMigrator.Migrate13to14(ctx)
```

Comment:
In `classic-terra/cosmos-sdk`, the `ConsensusVersion` of the `Staking` module is not auto-incremented in order to keep it consistent with `cosmos/cosmos-sdk`, so it is different from common module migrations.